src/Services/Manager.vala

```diff
@@ -59,6 +66,13 @@ public class BluetoothIndicator.Services.ObjectManager : Object {
         }
 
         retrieve_finished = true;
+
+        setup_dbus_rfkill.begin (() => {
+            if (killer == null) {
+                return;
+            }
```

Comment:
I do not see what this test does - there is no other code in the closure for when `killer != null` anyway??